fetch.md

```diff
@@ -1,11 +1,204 @@
+ - Nested browsing contexts and their subresources
+
+Concretely this can be done by overriding the [credentials mode][6] of all requests coming from a
+[prerendering browsing context][4] to the `"omit"` value.
+
+### Credentials After Activation
+
+After a [prerendering browsing context][4] is activated, requests originating from it would no longer have
+their credentials mode overridden, and would be fetched with whatever credentials were present in the user
+agent's cookie store as is typical on the web platform.
+
+> TODO: Discuss the possibility of partitioning the user agent's cookie store. In this proposal, the user
+agent effectively ignores cookies that it already has, for requests initiated from a
+[prerendering browsing context][4]. One downside of this proposal is that new credentials set on resources
+in a [prerendering browsing context][4] will entirely be ignored. If the user agent's cookie store was
+partitioned in a way similar to the HTTP cache, we could use an isolated speculative cookie store for all
+requests in a given [prerendering browsing context][4], and optionally merge cookies in some fashion after
+activation.
+
+## Stripping Referrer Information
+
+To preserve the user's privacy, there should be a limited amount of referrer information sent along with
+speculative navigation requests. Given the sensitive nature of these requests, we believe it should not
+be possible to expose the full referrer path when these requests are cross-origin. Instead, the total referrer
+information will be limited to at most the referrer's origin. If a developer supplies the `referrerpolicy`
+attribute on e.g., a `<link rel=prerender>` it should only be used for tightening the referrer policy beyond
+the platform [default][7] of `strict-origin-when-cross-origin`. We can achieve this by defining a list of
+a list of __sufficiently-strict referrer policies__ that are allowed for these requests.
+
+As mentioned, the list of sufficiently-strict referrer policies are the policies that are as or more
+strict compared to the platform [default][7] of `strict-origin-when-cross-origin`, which includes the
+following policies:
+ - `strict-origin-when-cross-origin`
+ - `same-origin`
+ - `strict-origin`
+ - `no-referrer`
+
+A speculative navigation request supplied with a referrer policy outside of this list will be silently
+cancelled.
+
+> TODO: We can also consider dispatching a console warning or firing the element's error event in these
+cases to let developers know that the request was not able to be dispatched.
+
+Furthermore, since the activation of a speculative navigation request keys on the referrer policy, we
+guarantee that there is never a discrepancy between the referrer sent with the request, and the value
+of `document.referrer` on the prerender page that may eventually get activated and promoted to using
+first-party storage. For an example of this, see [this issue & comment][8].
+
+### Referrer Information After Activation
```

Comment:
I don't think this is talking about after activation; it's talking about referrer information on subresource requests and nested browsing contexts. Right?

Reply:
Hmm, yeah that's right. My intention was to say there is no referrer information change on any requests after activation, because there is no referrer information change on any requests period, after the initial top-level request. I didn't really capture that though, so I'll do that now.